qa/tasks/ceph_test_case.py

```diff
@@ -18,6 +18,7 @@ class CephTestCase(unittest.TestCase):
     # Environment references
     mounts = None
     fs = None
+    recovery_fs = None
```

Comment:
> That intuitively makes more sense to me, too, but I preferred putting it next to the primary filesystem.

I believe `fs = None` should probably be in `cephfs_test_case.py` too. It is probably a holdover from when John broke `ceph_test_case.py` out of `cephfs_test_case.py`. @jcsp ?